docs/running_on_mainnet.md

```diff
@@ -1 +1,121 @@
-# TODO(const)
\ No newline at end of file
+# Running on the Main Network
+This tutorial shows how to use the bittensor `btcli` to create a subnetwork and connect your mechanism to it. It is highly recommended that you run `running_on_staging` and `running_on_testnet` first before attempting to register on mainnet. Mechanisms running on the main are open to anyone, emit real TAO, and incurs a `lock_cost` in TAO to create, and you should be careful not to expose your private keys, to only use your testnet wallet, not use the same passwords as your mainnet wallet, and make sure your mechanism is resistant to abuse. 
+
+
+## Steps
+
+1. Clone and Install Bittensor and the Bittensor Subnet Template.
+This clones and installs the template if you dont already have it (if you do, skip this step)
+```bash
+cd .. # back out of the subtensor repo
+git clone https://github.com/opentensor/bittensor-subnet-template.git # Clone the bittensor-subnet-template repo
```

Comment:
Maybe clarify here that people reading this have used the template to create their own repo and so `opentensor/bittensor-subnet-template` should be replaced with their repo name if that is the case